### DISCUSSION TITLE
How can I record a CAN log file of 2 more Vector channels?

### DISCUSSION THREAD

**Discussion by Dr-Br0t** [hardbyte/python-can#1665] [Q&A]
I am using a VN1630, and I want to record a CAN log of 2 or more channels, I know how to record data from only one channel, for this I use the example given example in the documentation. 

from can import Notifier, SizedRotatingLogger
from can.interfaces.vector import VectorBus

bus = VectorBus(channel=[0], app_name="CANape", fd=True)

logger = SizedRotatingLogger(
    base_filename="my_logfile.asc",
    max_bytes=5 * 1024 ** 2,  # =5MB
)
logger.rollover_count = 23  # start counter at 23

notifier = Notifier(bus=bus, listeners=[logger])

**Comment by zariiii9003** [comment]
You can either create a VectorBus instance with multiple channels: `VectorBus(channel=[0, 1], ...)` or you can add multiple bus instances to the Notifier, e.g. `Notifier(bus=[bus1, bus2], listeners=[logger])`

**Reply by Dr-Br0t** [reply]
Thank you, I tried that last Friday, I guess I did something bad, I will try with this, if I get an error I will post it here.
